### PR TITLE
create searchPAge

### PR DESCRIPTION
**Pull request by psi9730**

### CODE REVIEW

File: src/modules/main_page/mainPageSaga.js [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
-    password,
-  }
-
+function* requestGetCard() {
   try {
-    const token = yield api.post('http://127.0.0.1:8000/api-auth/', body
+    const token = yield api.get('http://127.0.0.1:8000/card/',
     )
     if (response) {
-      console.log(response)
```
Comment (SeoSeongHo) [inline src/modules/main_page/mainPageSaga.js]: erase

File: src/modules/main_page/mainPageView.js [hunk 2]
```diff
@@ -16,13 +16,13 @@ type State = {
...
 type Props = {
-  t: Function,
+  cards: any,
   loading: boolean,
-  onLoginPressed: (username: string, password: string) => void,
 };
 class MainPageView extends Component<Props, State> {
   constructor(props) {
     super(props);
+    console.log(this.props,"props");
```
Comment (SeoSeongHo) [inline src/modules/main_page/mainPageView.js]: erase

File: src/modules/main_page/mainPageView.js [hunk 3]
```diff
@@ -37,7 +37,8 @@ class MainPageView extends Component<Props, State> {
     };
   }
  componentDidMount() {
-    this.props.getCardListRequest().catch(e=>console.log(e));
+    this.props.getCardAllRequest().catch(e=>console.log(e));
+   console.log(this.props,"props");
```
Comment (SeoSeongHo) [inline src/modules/main_page/mainPageView.js]: erase

File: src/modules/search_page/SearchSaga.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+// import _ from 'lodash'
+
+const { API_ROOT } = Constants
+
+function* requestGetCard({query}) {
+  try {
+    const token = yield api.get(`${API_ROOT}/card/${query}`
+    )
+    if (response) {
+      console.log(response);
```
Comment (SeoSeongHo) [inline src/modules/search_page/SearchSaga.js]: erase

File: src/modules/search_page/SearchView.js [hunk 5]
```diff
@@ -12,9 +20,11 @@ class ContactInfo extends Component{
   }
 }
 
-class SearchView extends Component{
+export class SearchView extends Component<Props, State>  {
   constructor(props){
     super(props);
+    console.log(this.props,"props");
```
Comment (SeoSeongHo) [inline src/modules/search_page/SearchView.js]: erase

File: src/modules/search_page/SearchView.js [hunk 6]
```diff
@@ -23,6 +33,20 @@ class SearchView extends Component{
         {name: "dd", phone: "010-0000-0003"}
       ]
     }
+    console.log(this.state);
```
Comment (SeoSeongHo) [inline src/modules/search_page/SearchView.js]: erase

File: src/modules/search_page/SearchView.js [hunk 7]
```diff
@@ -23,6 +33,20 @@ class SearchView extends Component{
...
+  }
+
+  componentWillMount() {
+    const search = this.props.location.search
+    console.log(this.props,"props");
```
Comment (SeoSeongHo) [inline src/modules/search_page/SearchView.js]: erase

File: src/modules/top_navigator/navigatorSaga.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  const body = {
+    username,
+    password,
+  }
+
+  try {
+    const token = yield api.post('http://127.0.0.1:8000/api-auth/', body
+    )
+    if (response) {
+      console.log(response)
```
Comment (SeoSeongHo) [inline src/modules/top_navigator/navigatorSaga.js]: erase

File: src/utils/api/index.js [hunk 9]
```diff
@@ -17,7 +17,7 @@ export const checkStatus = (response) => {
 export const parseJSON = response => response.json()
 
  export const parseSettings = ({ method = 'get', data, locale, ...otherSettings } = {}) => {
-
+   console.log(data,"data");
```
Comment (SeoSeongHo) [inline src/utils/api/index.js]: erase

File: src/utils/api/index.js [hunk 10]
```diff
@@ -93,6 +93,7 @@ api.create = (settings = {}) => ({
   },
 
   post(endpoint, data, settings) {
+    console.log(data,"data");
```
Comment (SeoSeongHo) [inline src/utils/api/index.js]: erase

File: src/routes.js [hunk 11]
```diff
@@ -4,30 +4,26 @@ import App from 'components/App'
...
 import loginView  from './modules/login/loginViewContainer'
 import CardWriteView  from './modules/card_write/CardWriteViewContainer'
-import CardView  from './modules/main_page/mainPageView'
+import CardView  from './modules/main_page/mainPageViewContainer'
 import UserProfileView  from './modules/user_profile/UserProfileViewContainer'
 import PetProfileView  from './modules/pet_profile/PetProfileViewContainer'
 import CardDetailView from './modules/card_detail/CardDetailViewContainer'
 import SignUpView from 'modules/login/SignUpViewContainer'
-import SearchView from 'modules/search_page/SearchView'
+import SearchViewContainer from 'modules/search_page/SearchViewContainer'
```
Comment (SeoSeongHo) [inline src/routes.js]: 앞에는 serachview아님? import SearchView from ~~

Comment (psi9730) [reply src/routes.js]: 뭐라부르든 상관없음
밑에만 잘넣어주면됌

**Pull request merged**